src/main/resources/content/k8s/do-not-allow-volume-claims-to-be-read-by-many-nodes.json

```diff
@@ -1 +1,8 @@
-{"severity":"low","rule":"$.spec.volumeClaimTemplates[*].spec.accessModes == ReadOnlyMany","id":"802f2ed9-0b0d-4627-bf1a-7cb0ccfdd71c","enabled":true,"resourceType":"k8s","policy":"do not allow volume claims to be read by many nodes"}
\ No newline at end of file
+{
+  "severity": "low",
+  "rule": "$.spec.volumeClaimTemplates[*].spec.accessModes == ReadOnlyMany",
+  "id": "802f2ed9-0b0d-4627-bf1a-7cb0ccfdd71c",
+  "enabled": true,
+  "resourceType": "k8s",
+  "policy": "do not allow volume claims to be read by many nodes"
```

Comment:
description field missing here as well.
Also, the policy name is not sentence case.